### ISSUE TITLE
Does not work on Plasma 5.19 Beta

### ISSUE THREAD

**Issue by DanielRios549** [zzag/kwin-effects-yet-another-magic-lamp#26]
I recently update my KDE Neon to 5.19 Beta from Testing repo (I change in neon.list), and the effect stop working, System Settings shows it as an unsupporterd effects, hiden by default, and even when selected, it does not work.

![image](https://user-images.githubusercontent.com/11152535/82344537-8b6f6b80-99ca-11ea-9a4b-3be2b33ac415.png)

**Comment by zzag** [comment]
> I recently update my KDE Neon to 5.19 Beta

You need to recompile/re-install this effect.

**Comment by Thyrux** [comment]
```
-- Could NOT find kwineffects_kwineffects (missing: kwineffects_kwineffects_LIBRARY kwineffects_kwineffects_INCLUDE_DIR) (found version "")
-- Could NOT find kwineffects_kwinglutils (missing: kwineffects_kwinglutils_LIBRARY kwineffects_kwinglutils_INCLUDE_DIR) (found version "")
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  Could NOT find kwineffects (missing: kwineffects_LIBRARIES kwineffects kwinglutils)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/Findkwineffects.cmake:95 (find_package_handle_standard_args)
  CMakeLists.txt:35 (find_package)
```

**Comment by Thyrux** [comment]
> You need to recompile/re-install this effect.

Unfortunately it doesn't help.

**Comment by michaldybczak** [comment]
I also have this issue. I have AUR version of this plugin. I recompiled it but it's still not working and shows as empty on the list.

Maybe AUR version is obsolete?

https://aur.archlinux.org/packages/kwin-effects-yet-another-magic-lamp

**Comment by zzag** [comment]
No, the AUR version is fine, pkgrel can be bumped though. I suspect it's a cache related issue on your side.

**Comment by michaldybczak** [comment]
Ah, you are right. After reboot, the effect shows and I can set it, it works. I gave up on non-breeze aurorae themes so your plugin would work :P. The crashes were too tiring, plus the change is nice. Still, I remember about the crashing issue with third party aurorae themes. Too bad it requires kwin recompilation and that is what I'm afraid and that is why I haven't done this yet.

**Comment by michaldybczak** [comment]
After upgrading to Plasma 5.20 the effect also stopped working. I recompiled it again and it's working.
I believe this topic can be closed since 5.19 is already an old version, plus the issue here has a pretty regular fix.
As long someone is not officially packaging it but it's installed directly or from AUR, this has to be recompiled with bigger Plasma updates.
